rubicon/objc/runtime.py

```diff
@@ -2097,14 +2105,21 @@ def __init__(self, func, restype=NOTHING, *arg_types):
 
         self.literal = BlockLiteral()
         self.literal.isa = addressof(_NSConcreteGlobalBlock)
-        self.literal.flags = BlockConsts.HAS_STRET | BlockConsts.HAS_SIGNATURE
+        self.literal.flags = BlockConsts.HAS_STRET | BlockConsts.HAS_SIGNATURE | BlockConsts.HAS_COPY_DISPOSE
         self.literal.reserved = 0
         self.cfunc = self.cfunc_type(self.wrapper)
         self.literal.invoke = cast(self.cfunc, c_void_p)
         self.descriptor = BlockDescriptor()
         self.descriptor.reserved = 0
         self.descriptor.size = sizeof(BlockLiteral)
 
+        self.copy_helper_type = CFUNCTYPE(c_void_p, c_void_p, c_void_p)
+        self.dispose_helper_type = CFUNCTYPE(c_void_p, c_void_p)
```

Comment:
Minor thing: since the signatures of the helpers are always the same, these `CFUNCTYPE`s could be global constants instead of attributes. This would also allow setting the correct types for the `BlockDescriptor` struct's `copy_helper` and `dispose_helper` fields.

Reply:
Ah I see.  Yeah no sense in keeping that data around in the instance when it's always the same.  Got it.